<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synthetical Acoustic System Debugger</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="curveWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outputButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="parameterPanelWidget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>301</y>
      <width>361</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Choose a synchrony method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Choose a synchrony method</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>curveWidget</zorder>
   <zorder>outputButton</zorder>
   <zorder>parameterPanelWidget</zorder>
   <zorder>logoLabel</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
